File: 02Fabryka/Students/2018/DawidowskiMarcin/src/SimpleFactory/Car.java
package SimpleFactory;

public abstract class Car {

    protected String brandName;
    protected String model;
    protected String type;
    protected String fuelType;
    protected int price;

    public void carDetails() {
        System.out.println("=======================================");
        System.out.println("Brand: " + brandName);
        System.out.println("Model: " + model);
        System.out.println("Type: " + type);
        System.out.println("Type: " + fuelType);
        System.out.println("Type: " + price);

        System.out.println("Creating a car.");
    }
}

File: 02Fabryka/Students/2019/KucmanBłażej/FactoryPattern/FactoryPattern/SimpleFactory/Cars/Car.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FactoryPattern.SimpleFactory.Cars
{
    public abstract class Car : IWehicle
    {
        public string Name { get; set; }
        public int Price { get; set; }
        public int Speed { get; set; }
        public int Horsepower { get; set; }

        public int CapacityPeople { get; set; }

        public void getSound()
        {
            Console.WriteLine("brum");
        }
    }
}
